acsoo/makepot_script

```diff
@@ -4,9 +4,16 @@
 
 import base64
 import os
+import re
 import subprocess
 from os.path import join
 
+
+LINE_PATTERNS_TO_REMOVE = [
+    r'\"POT-Creation-Date:.*?\"[\n\r]',
+    r'\"PO-Revision-Date:.*?\"[\n\r]',
```

Comment:
do we need \" or is " enough?

Reply:
No indeed... ```"``` is enough. The online tester I used forced me to escape this character.